src/pages/doc/user-guide/glossary.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
```

Comment:
Basic concepts pages are giving 404s. For example, the route [`doc/user-guide/basic-concepts/dvc-project`](https://dvc.org/doc/user-guide/basic-concepts/dvc-project) is leading to a 404 though it should exist 🤔 

https://dvc-org-try-external-th-doe8zu.herokuapp.com/doc/user-guide/basic-concepts/dvc-project

Reply:
The glossary page seems to be generated from basic concepts. So, that must be okay. I can see it on https://dvc-org-try-external-th-doe8zu.herokuapp.com/doc/user-guide/glossary

Reply:
Totally forgot we made these pages, I think the only thing stopping these pages from being made is an early return in the theme code that I put in thinking about the site before we made these into pages. Thanks!

Reply:
Fixed these with a published change to the theme. Maybe we can make it configurable in the future, but we won't use it internally so we can cross that bridge when we come to it later.